src/test/java/seedu/address/testutil/EventBuilder.java

```diff
@@ -15,13 +21,6 @@
  */
 public class EventBuilder {
 
-    public static final String DEFAULT_EVENT_DESCRIPTION = "CS2101 Tutorial";
-    public static final String DEFAULT_DATE = "2022-03-12";
-    public static final String DEFAULT_TIME = "14:00";
-    public static final String DEFAULT_DURATION = "PT2H";
-
-    private static final RecurFrequency DEFAULT_RECURRENCE = null;
```

Comment:
I think we should keep these here, instead of relying on 2 other classes. It is also easier to reference this class while testing instead of referencing 2 other classes.

Reply:
Hmm, I was thinking that it would make the code less maintainable, since in the future if we want to change a default value we need to change at 2 different places.

Reply:
In that case would it be better to  rename it VALID_EVENT_DESCRIPTION etc rather than DEFAULT_EVENT_DESCRIPTION etc?

Reply:
I did this while following PersonBuilder. DEFAULT was used there so I think that it is fair to use it here as well. Perhaps the visibility should change instead. And also I think that this edit should have been done when I committed the file.